tests/monster_test.fbs

```diff
@@ -15,13 +15,13 @@ namespace MyGame.Example;
 attribute "priority";
 
 /// Composite components of Monster color.
-enum Color:ubyte (bit_flags) { 
```

Comment:
@CasperN Is this just whitespace changes?

Reply:
Yes